static/_redirects

```diff
@@ -1,5 +1,7 @@
 # Redirects
 
+/support/how-to/rackconnect-regionlink-faq/  /how-to/rackconnect-regionlink-faq/  404
+/how-to/rackconnect-regionlink-faq/  /support/rackconnect-regionlink-faq/  301
```

Comment:
Same issue here, see the retired articles section on line 41.

As formatted, this redirect is going to result in a 404